build.sbt

```diff
@@ -37,13 +37,12 @@ lazy val common = project
       "com.typesafe" % "config" % "1.3.1",
       "org.joda" % "joda-convert" % "1.8.1",
       "org.typelevel" %% "cats" % "0.9.0",
-      "com.typesafe.scala-logging" % "scala-logging_2.11" % "3.4.0",
+      "com.typesafe.scala-logging" % "scala-logging_2.12" % "3.5.0",
```

Comment:
This should change to `"com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",`